src/views/Home.jsx

```diff
@@ -1,8 +1,38 @@
 import './Home.css';
 import { SingleList } from '../components';
+import { useState } from 'react';
+import { createList, useAuth } from '../api';
+import { useNavigate } from 'react-router-dom';
 
 export function Home({ data, setListPath }) {
 	const hasList = data.length !== 0;
+
+	const { user } = useAuth();
+	const userId = user?.uid;
+	const userEmail = user?.email;
+
+	const [inputValue, setInputValue] = useState('');
+	const navigate = useNavigate();
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+
+		try {
+			await createList(userId, userEmail, inputValue);
```

Comment:
suggestion: I know Ross also mentioned using `react-hot-toast` for the alert and with your suggestion on ours about clearing the form when done I looked more in how to use the toast to handle the success and the error. so if yall want to also use it I think it would look something like this for yalls call

```suggestion
			await toast.promise(
				createList(userId, userEmail, inputValue),
				{
					pending: , // a message about adding new list
					success: () => {
						const path = `${user.uid}/${inputValue}`;
                        setListPath(path);
                        setInputValue('');
                        navigate('/list');
                        return 'Success: Your new list is created!';
					},
					error: () => {
						return 'Failed to create the list. Please try again.';
					},
				},
				{
					style: {
						minWidth: '250px',
					},
				},
			);
```

Reply:
Thanks for the suggested code Brianna! We tried a tad simpler approach for now (as shown below), and can definitely refactor if needed in the future. 
<img width="572" alt="Screenshot 2024-08-25 at 9 08 07 AM" src="https://github.com/user-attachments/assets/7e476dcc-f565-47cd-8b32-64087f6fa785">